packages/components/src/layout/docs/inline/migration-notes.md

```diff
@@ -0,0 +1,3 @@
+Coming from Orbiter, you should be aware of the following change:
+
+- The default `gap` value has been reduced to `1rem` / `var(--hop-space-inline-md)` from `1.25rem`. Use `UNSAFE_gap="1.25rem"` as a temporary measure during the migration to the Hopper design language. This value should eventually be updated to `1rem` / `var(--hop-space-inline-md)` to align with the new design standards.
```

Comment:
should it be "Hopper design system" instead or language?
Also, I'm not sure we should so that the value should be updated to 1rem. They won't be updating the UNSAFE_gap, they will be removing it and the result will be them using what's in hopper.

Reply:
I purposefully used design language, but system works and might be more geared toward the devs.

Since we wan't to help with migration, I thought we'd want do let the user know. That said it's such a tiny difference that we might just let them know, otherwise we'd should not even add this to the migration notes, as it'll be "seamless".

Reply:
well I do think we should tell them what to add and then tell them that it can be removed to follow the design system.

Reply:
Which is what I'm doing no/

Reply:
alright, just suggestions :P